tests/Predis/Command/RawCommandTest.php

```diff
@@ -60,44 +60,13 @@ public function testStaticCreate()
 
     /**
      * @group disconnected
-     * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage The arguments array must contain at least the command ID.
      */
     public function testExceptionOnMissingCommandID()
     {
-        new RawCommand(array());
-    }
+        $this->expectException('InvalidArgumentException');
+        $this->expectExceptionMessage('The arguments array must contain at least the command ID');
 
-    /**
-     * The signature of RawCommand::create() requires one argument which is the
-     * ID of the command (other arguments are fetched dinamically). If the first
-     * argument is missing, PHP emits an E_WARNING.
-     *
-     * @group disconnected
-     */
-    public function testPHPWarningOnMissingCommandIDWithStaticCreate()
-    {
-        if (version_compare(PHP_VERSION, "7.1", '>')) {
-            $this->markTestSkipped('only for PHP < 7.1');
-        }
-        $this->setExpectedException('PHPUnit_Framework_Error_Warning');
-        RawCommand::create();
-    }
-
-    /**
-     * The signature of RawCommand::create() requires one argument which is the
-     * ID of the command (other arguments are fetched dinamically). If the first
-     * argument is missing, PHP 7.1 throw an exception
-     *
-     * @group disconnected
-     */
-    public function testPHPWarningOnMissingCommandIDWithStaticCreate71()
-    {
-        if (version_compare(PHP_VERSION, "7.1", '<')) {
-            $this->markTestSkipped('only for PHP > 7.1');
-        }
-        $this->setExpectedException('ArgumentCountError');
-        RawCommand::create();
```

Comment:
The test `testPHPWarningOnMissingCommandIDWithStaticCreate71()` should be ket, just renamed into `testPHPWarningOnMissingCommandID()` and updated since our minimum target is now PHP 7.2 (meaning that we don't need the version check to mark the test as skipped when).